docs/src/main/tut/faq.md

```diff
@@ -28,15 +28,33 @@ import cats.implicits._
 
 This should be all that you need, but if you'd like to learn more about the details of imports than you can check out the [import guide](imports.html).
 
-## Why can't the compiler find implicit instances for Future?<a id="future-instances" href="#future-instances"></a>
+## <a id="future-instances" href="#future-instances"></a>Why can't the compiler find implicit instances for Future?
 
 If you have already followed the [imports advice](#what-imports) but are still getting error messages like `could not find implicit value for parameter e: cats.Monad[scala.concurrent.Future]` or `value |+| is not a member of scala.concurrent.Future[Int]`, then make sure that you have an implicit `scala.concurrent.ExecutionContext` in scope. The easiest way to do this is to `import scala.concurrent.ExecutionContext.Implicits.global`, but note that you may want to use a different execution context for your production application.
 
-## How can I turn my List of `<something>` into a `<something>` of a list?<a id="traverse" href="#traverse"></a>
+## <a id="traverse" href="#traverse"></a>How can I turn my List of `<something>` into a `<something>` of a list?
 
 It's really common to have a `List` of values with types like `Option`, `Xor`, or `Validated` that you would like to turn "inside out" into an `Option` (or `Xor` or `Validated`) of a `List`. The `sequence`, `sequenceU`, `traverse`, and `traverseU` methods are _really_ handy for this. You can read more about them in the [Traverse documentation]({{ site.baseurl }}/tut/traverse.html).
 
-## How can I help?<a id="contributing" href="#contributing"></a>
+## <a id="simulacrum" href="#simulacrum"></a>What does `@typeclass` mean?
+
+Cats defines and implements numerous type classes. Unfortunately, encoding these type classes in Scala can incur a large amount of boilerplate. To address this, [Simulacrum](https://github.com/mpilquist/simulacrum) introduces `@typeclass`, a macro annotation which generates a lot of this boilerplate. This elevates type classes to a first class construct and increases the legibility and maintainability of the code. Use of Simulacrum also ensures consistency in how the type classes are encoded across a project. Cats uses Simulacrum wherever possible to encode type classes, and you can read more about it at the [project page](https://github.com/mpilquist/simulacrum).
+
```

Comment:
> Cats uses Simulacrum wherever possible to encode type classes

We have made a conscious decision not to use it within cats-kernel to ensure backwards compatibility. I'm not sure whether or not that's worth bringing up here.